app/components/message-box.js

```diff
@@ -10,6 +10,7 @@ export default Ember.Component.extend({
   btn2Text: "",
   btn2Callback: () => {},
   displayCloseLink: false,
+  router: Ember.inject.service('-routing'),
```

Comment:
Why there is `-` in routing service??

Reply:
Using router service to access current route.